Dockerfile

```diff
@@ -63,10 +57,10 @@ ADD . $APPDIR
 # then we want to link the source (with the -e flag) and if we're in prod, we
 # want to delete the stuff in the /code folder to keep it simple.
 RUN if [ "$ENVIRONMENT" = "deployment" ] ; then\
-        pip install .[$ENVIRONMENT] ; \
+        pip install .[$ENVIRONMENT] --extra-index-url https://packages.dea.ga.gov.au/; \
```

Comment:
Does it make sense for an ODC repo to depend on DEA infra?

Reply:
Its being installed regardless, see `requirement-docker.txt`, this pr is simplify removing that `.txt` and put them together